src/com/owncloud/android/lib/resources/shares/ShareType.java

```diff
@@ -75,7 +76,8 @@ public static ShareType fromValue(int value)
             return CONTACT;
         case 6:
             return FEDERATED;
+        default:
+            return NO_SHARED;
         }
-        return null;
     }
 };
```

Comment:
New line please.

Reply:
instead of the removed line?

Reply:
No, after };

Reply:
Done via https://github.com/nextcloud/android-library/pull/89/commits/b0ebaae55bc3cea88bfa15ce0081b285c2d9f0e3